main.js

```diff
@@ -11,15 +11,21 @@
       this.elementId = elementId
     }
 
+    
     add(title){
       this.createCheckbox(title)
-      this.todos.push({id: 2, title: title, isCompleted: false})
+      getMaxId()
+      const obj = {id: maxId + 1, title: title, isCompleted: false}
```

Comment:
getMaxId `最大のIDを取得` なので目的語がなく、中身を読みにいかないといけない感じになります。

こうするとコードから意味が伝わるかと。

```suggestion
      const task = {id: generateTaskId(), title: title, isCompleted: false}
```